librz/core/cmd.c

```diff
@@ -4760,55 +4764,130 @@ DEFINE_HANDLE_TS_FCN_AND_SYMBOL(redirect_command) {
 	return res;
 }
 
+typedef struct _search_help {
+	bool color;
+	PJ *pj;
+} RzHelpSearch;
+
+static bool do_help_manual(RzCmd *cmd, const char *cmdname, bool color) {
+	if (!cmdname) {
+		return false;
+	}
+	char *cmdname_help = NULL;
+	char *help_msg = NULL;
+	RzCmdParsedArgs *help_pra = NULL;
+
+	cmdname_help = rz_str_newf("%s?", cmdname);
```

Comment:
maybe we can think about a new API `rz_cmd_desc_get_help` that directly takes a RzCmdDesc instead of a RzCmdParsedArgs.